### ISSUE TITLE
Sympa Accessibility Issue - 9 - Data cells in complex data tables must use HEADERS attribute to identify header cells.

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1767]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
The data cells in complex data tables need to use the HEADERS attribute to identify the appropriate header cells so the screen reader properly navigates through the table.

Actual behavior
---------------
Data cells in data tables do not have a HEADERS attribute to identify header cells.   The screen reader is getting confused.

Steps to reproduce
------------------
Sympa Accessibility Issue - 9 - was determined by an Automation Tool Evaluation using AInspector and Siteimprove Accessibility Checkers as automated accessibility evaluation tools.

After running these automation tools the following issue was identified:

Data cells in complex data tables must use HEADERS attribute to identify header cells. The data cells in complex data tables need to use the HEADERS attribute to identify the appropriate header cells, since simple row/column relationships cannot be relied upon to provide header information.

[WCAG Level A 1.3.1 Info and Relationships (Primary)](https://www.w3.org/TR/WCAG21/#info-and-relationships)

![image](https://github.com/sympa-community/sympa/assets/144282683/b6218b3b-4ea0-466e-a773-a6330adf244b)

Fig 10 Data cells must have headers.

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13549960/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)
